backend/tests/utils.test.js

```diff
@@ -0,0 +1,21 @@
+const { sortAlphaNum } = require('../utils');
+
+describe('sortAlphaNum', () => {
+  it('should sort list correctly', () => {
+    const data = [
+      'WORKSHOP 11',
```

Comment:
Maybe include a single digit, and other strings without digits, or a complex mixture?